src/dsp_tools/utils/connection_live.py

```diff
@@ -11,14 +11,15 @@
 import regex
 from requests import ReadTimeout, RequestException, Response, Session
 
-from dsp_tools.models.exceptions import BadCredentialsError, BaseError, PermanentConnectionError, UserError
+from dsp_tools.models.exceptions import BadCredentialsError, BaseError, InputError, PermanentConnectionError, UserError
 from dsp_tools.utils.create_logger import get_logger
 from dsp_tools.utils.set_encoder import SetEncoder
 
 HTTP_OK = 200
 HTTP_UNAUTHORIZED = 401
 
 logger = get_logger(__name__)
+LOGFILES = ", ".join([handler.baseFilename for handler in logger.handlers if isinstance(handler, FileHandler)])
```

Comment:
IMO this should be a function somewhere in a util, preferably in a `log_util` together with `get_logger()`. It is very low level and has nothing to do with connections

Reply:
I agree that it is not nice. so you would do something like this?

```
from log_util import get_log_info

logger, logfiles = get_log_info()
```

Reply:
I'd rather keep them separate:

```python
from log_util import get_logger, get_log_files

logger = get_logger(__name__)
logfiles = get_log_files()
```

but by all means, this should not block this PR